### PR TITLE
fix #117 make homepage wysiwygly editable from admin panel

### LINKED ISSUE #117: Add editable HomePage model
To allow core-members to dynamically edit the content of the home page:
- title (Welcome)
- image (Banner)
- description (LFD about)
- volunteer instructions (rendered as |safe)
- pickleader instructions (rendered as |safe)

See also FAQ in issue #29

### PR DESCRIPTION
**Pull request by 2ynn**
Addresses #117 

- added `Content` model with title/subtitle/body fields (en/fr)
- installed `django-quill` for WYSIWYG editing

screenshot_2024-04-08_00:54:07
screenshot_2024-04-08_00:53:56

### CODE REVIEW

File: saskatoon/sitebase/views.py [hunk 1]
```diff
@@ -7,11 +7,29 @@
...
+
+        content_name = VOLUNTEER_HOME_CONTENT_NAME
+
+        if self.request.user.is_authenticated:
+            content_name = PICKLEADER_HOME_CONTENT_NAME
+
+        home = Content.objects.get(name=content_name)
+        if home is not None:
+            context['content'] = home.content(self.request.LANGUAGE_CODE)
+
```
Comment (2ynn) [inline saskatoon/sitebase/views.py:30]: TODO: pass a dummy content object here if there are no appropriate object saved in db

File: saskatoon/sitebase/templates/app/index.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {% block content %}
 
 <div class="breadcomb-area">
-    <div class="container mg-t-30">
+    <div class="container">
```
Comment (2ynn) [inline saskatoon/sitebase/templates/app/index.html:8]: TODO: check if `mg-t-30` class is needed

File: saskatoon/sitebase/views.py (hunk 1 above)
Comment (oxgr) [reply saskatoon/sitebase/views.py:30]: my lsp is saying `Content.objects` doesn't exist. as it stands, hitting `/` route gives this error. wrapping the method in try/catch and manually setting `home = None` on exception at least bypasses this, but original problem remains and `get()` may never get called. not sure where to go from there.

Comment (2ynn) [reply saskatoon/sitebase/views.py:30]: have you run migrations? `saskatoon/manage.py migrate`

Comment (2ynn) [reply saskatoon/sitebase/views.py:29]: pushed new commit, now if the `Content` object with reference name `pickleader_home`/`volunteer_home` doesn't exist when first accessing the home page, a blank object is created.

**Comment by oxgr** [comment]
loads good with fresh db and fixtures! looks good to go?

**Review by oxgr — APPROVED** [review]

**Pull request merged**